python/Ganga/Runtime/Repository_runtime.py

```diff
@@ -134,7 +134,15 @@ def bootstrap():
         registry.startup()
         logger.debug("started " + registry.info(full=False))
         if registry.name == "prep":
-            registry.print_other_sessions()
+            # Get a list of strings describing other running sessions
+            other_sessions = registry.get_other_sessions()
```

Comment:
Again, I _really_ want to avoid calling the repository from in front of the registry. We shouldn't know anything about the internals of the registry in order to do things like this. We potentially never needed the pint function in this case but it makes sense to have this distinction here I think.